music/src/core/player.ts

```diff
@@ -78,13 +79,25 @@ export abstract class BasePlayer {
    *   @param callbackObject An optional BasePlayerCallback, specifies an
    *     object that contains run() and stop() methods to invode during
    *     playback.
+   *   @param qpm An optional number specifying the starting BPM for playback.
    */
-  constructor(playClick = false, callbackObject?: BasePlayerCallback) {
+  constructor(playClick = false, callbackObject?: BasePlayerCallback,
+              qpm = 60) {
```

Comment:
I don't think we should specify a default qpm in the constructor. Instead, we should play the sequence back at the qpm in given as an argument to `start()`, the desired QPM if `setTempo` is called and no argument is given, the NoteSequence tempos[0].qpm if no argument is given and `setTempo` hasn't been called, or the DEFAULT_QUARTERS_PER_MINUTE if none of the above.

Reply:
done.